.github/workflows/ci_workflows.yml

```diff
@@ -95,6 +95,9 @@ jobs:
         python-version: ${{ matrix.python }}
     - name: Install base dependencies
       run: python -m pip install --upgrade pip tox
+    - name: Install extra dependencies for strauss
+      if: "contains(matrix.toxenv, 'alldeps')"
+      run: sudo apt-get install libportaudio2
```

Comment:
will this be an issue for users as well?  Should we catch the `OSError` along with the `ImportError` in cubeviz/viewers?

Reply:
I see there are docs updates, but if a user uses `all` but does not manually install `libportaudio2`, then they'll get an `OSError` when importing jdaviz, right?  Do we either want to gracefully ignore that and keep the plugin disabled with a message or give a more instructive error instead?  Or is the error message obvious enough that people will know how to address it?

Reply:
ok, we can always add catches later if people run into problems - thanks!